### ISSUE TITLE
Use tabs instead of spaces

### ISSUE THREAD

**Issue by jhnns** [peerigon/eslint-config-peerigon#76]
Since we're using Prettier anyway (see also #74), we could also switch from spaces to tabs. **The main advantage I see here is that developers could configure their preferred size of indentation.**

GitHub used to display tabs with 8 spaces which was pretty ugly but that's not the case anymore. I've played around with tabs in parse-domain, you can check out this [draft PR](https://github.com/peerigon/parse-domain/compare/master...next) to see tabs in action.

What do you think?

**Comment by bytetyde** [comment]
Some file formats can't handle tabs, e.g yaml files go haywire when the include tabs. 
We maybe should check what file formats should still use spaces and exclude them.

Currently I'm only aware of yaml files because that already brought me some headaches, but maybe there are some more. At least worth a quick search/thoughts.

**Comment by jhnns** [comment]
Is Prettier using tabs in yaml files? Afaict it's strongly discouraged to use tabs, so I would expect Prettier to ignore the tab setting ^^

**Comment by mksony** [comment]
@jhnns maybe I am missing something but for me github still shows 8 spaces for tabs as default. The only way I found to change this was to set a parameter which I found here https://stackoverflow.com/a/33831598 e.g. `?ts=4` which gets lost after reload.

Other than that people say that github respects the tab size configured in `.editorconfig` which means we can't really solve the problem on eslint level.

Any ideas how to solve that? For me it seems that spaces provide better defaults out of the box, and tabs could be something where we need additional configuration to make it work if a team decides to use them.

**Comment by mksony** [comment]
@jhnns the default for `eslint-config-peerigon/prettier` seems to be tabs already, so I am wondering what this issue here is about since it is already implemented.

**Comment by mksony** [comment]
Another thought against the tab default seems to be that the default `package.json` indentation is 2 spaces. So I would argue that the best out of the box default would be 2 spaces and everything else can be changed in userland.

**Comment by jhnns** [comment]
> @jhnns maybe I am missing something but for me github still shows 8 spaces for tabs as default. The only way I found to change this was to set a parameter which I found here stackoverflow.com/a/33831598 e.g. ?ts=4 which gets lost after reload.

I found out that it's the [Refined GitHub browser extension](https://github.com/sindresorhus/refined-github#code). What a shame! Tab size 8 is ridiculous 😢
I personally started to like tabs, but this is not worth the effort.

I decided to remove the Peerigon Prettier configuration entirely. We should stick with whatever Prettier uses as default. Initially, I kind of wanted to preserve the Peerigon style (e.g. 4 spaces, double quotes) in order to reduce the noise in current projects. But with more team members coming to Peerigon we have more strong opinions about this and I think we all agree that these discussions are not worth it.

We should all resist configuring Prettier based on our personal preference.
